admin/cleanup.py

```diff
@@ -0,0 +1,379 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Utilities for cloud resource cleanup.
+"""
+from datetime import datetime, timedelta
+import json
+import sys
+from uuid import UUID
+import yaml
+from yaml.parser import ParserError
+
+from characteristic import attributes
+
+from dateutil.parser import parse as parse_date
+from dateutil.tz import tzutc
+
+from libcloud.compute.providers import get_driver, Provider
+
+from twisted.internet.defer import succeed
+from twisted.python.filepath import FilePath
+from twisted.python.log import err
+from twisted.python.usage import Options, UsageError
+
+# Marker value defined by ``flocker.testtools.cluster_utils.MARKER``.  This
+# should never change and should always identify test-created clusters.
+from flocker.testtools.cluster_utils import MARKER
```

Comment:
Looks like @brendan-donegan 's working on this from a different angle:
 * https://clusterhq.atlassian.net/browse/FLOC-3981

I guess we should swap notes.